csi-plug-n-play/sidecars/soda-csi-provisioner/external-provisioner/cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -156,11 +156,13 @@ func main() {
 	}
 
 	// Autodetect provisioner name
-	provisionerName, err := ctrl.GetDriverName(grpcClient, *operationTimeout)
+	//SODA-CHANGE
+	/*provisionerName, err := ctrl.GetDriverName(grpcClient, *operationTimeout)
 	if err != nil {
 		klog.Fatalf("Error getting CSI driver name: %s", err)
-	}
-	klog.V(2).Infof("Detected CSI driver %s", provisionerName)
+	}*/
+	provisionerName := "soda-csi-block"
```

Comment:
I think name can be changed as we already have block and file plugins

Reply:
All the current provisioner names in nbp/csi contains `opensds` in it's name, so I have kept this so that from here on we can maintain soda as the provisioner name.
Also we need to keep the name consistent in csi-plugins as well as csi-plug-n-play.

Reply:
ok

Reply:
@sushanthakumar updated the code